MAUI/MauiAppWithBroker/MSALClient/PCAWrapper.cs

```diff
@@ -86,6 +113,15 @@ internal async Task SignOutAsync()
             }
         }
 
+        /// <summary>
+        /// Gets scopes for the application
+        /// </summary>
+        /// <returns>An array of all scopes</returns>
+        internal string[] GetScopes()

```

Comment:
Brokered auth must try to get SSO with OS account. See how to do it in this sample. 

https://github.com/bgavrilMS/MSAL-SingleAccount-SSO

Reply:
Just to be sure, this would only apply with `WinUI` apps right?

Reply:
@SameerK-MSFT

Would you know if logging in with the Authenticator app is the same on Android and iOS? I don't believe there is a "default OS" account in Android/iOS but would caching work with the Authenticator app or is that redundant?